### PR TITLE
Added function to check if a country supports validation

### PR DESCRIPTION
**Pull request by EricStG**
#### What is this PR for?
This adds a method to check if a validation exists for a country.
This will enable us to determine whether a specific country code supports validation or not before attempting it.

#### Who should review this PR?
@melwynfurtado 

#### Questions:
- [X] All unit tests executed successfully in CI environment
- [X] Related tests executed successfully in CI environment
- [X] Documentation updated accordingly [e.g. Readme.md, Contributing.md]
- [X] PR ready for merging

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## next?
+* [feature] Added fu nction to support checking if validation exists for a specified country
```
Comment (melwynfurtado) [inline CHANGELOG.md]: fix typo please

File: test/main.spec.ts [hunk 2]
```diff
@@ -40,6 +40,29 @@ describe('postcodeValidator', () => {
...
     });
 
+    test('should throw error for invalid country codes', () => {
+        expect.assertions(1);
+        expect(() => postcodeValidator('SW1A 0AA', 'GB')).toThrow('Invalid country code: GB');
+    });
+});
+
+describe('postcodeValidatorExists', () => {
+    test('should return true for existing validators', () => {
```
Comment (melwynfurtado) [inline test/main.spec.ts]: Can you please also add a negative scenario for `postcodeValidatorExists` ?

File: CHANGELOG.md (hunk 1 above)
Comment (EricStG) [reply CHANGELOG.md]: Oops, fixed

File: test/main.spec.ts (hunk 2 above)
Comment (EricStG) [reply test/main.spec.ts]: Done

**Pull request merged**